lightning/src/routing/router.rs

```diff
@@ -602,6 +602,17 @@ fn compute_fees(amount_msat: u64, channel_fees: RoutingFees) -> Option<u64> {
 	}
 }
 
+/// The default `features` we assume for a node in a route, when no `features` are known about that
+/// specific node.
+///
+/// Default features are:
+/// * variable_length_onion_optional
+fn default_node_features() -> NodeFeatures {
```

Comment:
Decided to also name this function to `default_node_features`, so that it possibly can be extended with more features in the future. I also considered if it would make more sense to add this as part of the `NodeFeatures` type itself, but figured that it probably belongs to the router context.

Reply:
I like this refactoring :)

Reply:
Thanks!